target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/xrx330.dtsi

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
lantiq uses files instead of files-5.4 now. Apart from that, I've recently moved the dts-v1 statement to the dtsi files; so, I'd be happy if you could move it from DTS to DTSI as well.

Reply:
BTW GPL-2.0+ is deprecated, the proper term is GPL-2.0-or-later